pkg/trace/agent/normalizer.go

```diff
@@ -194,6 +195,19 @@ func setChunkAttributesFromRoot(chunk *pb.TraceChunk, root *pb.Span) {
 		// Older tracers set origin in the root span.
 		chunk.Origin = root.Meta[tagOrigin]
 	}
+
+	if _, ok := chunk.Tags[tagDecisionMaker]; !ok {
+		var set bool
+		for _, span := range chunk.Spans {
+			if dm, ok := span.Meta[tagDecisionMaker]; !set && ok {
+				if chunk.Tags == nil {
```

Comment:
Is possible that `chuck.Tags` is nil here? In line 199 we are doing `chunk.Tags[tagDecisionMaker]` if `chuck.Tags` is nil it would cause a panic, right?

Reply:
The condition is
```
	if _, ok := chunk.Tags[tagDecisionMaker]; !ok {
	```
And `!ok` will be true if `chunk.Tags` is `nil`, entering the branch. 

I actually saw a `nil` `chunk.Tags` during testing, which prompted this check.

Reply:
Ohh you are right is `!ok` so it does makes sense that `chunk.Tags` is nil 😄 the first time